src/app/client/src/app/modules/groups/components/group-form/group-form.component.html

```diff
@@ -1,55 +1,58 @@
 <!-- add create group form -->
-<!-- <sui-modal *ngIf="showBoardSelectnModal" [mustScroll]="true" [isClosable]="true" [transitionDuration]="0"
-  [size]="'normal'" class="sb-modal" appBodyScroll (dismissed)="showBoardSelectnModal = !showBoardSelectnModal" #modal>
+<sui-modal [mustScroll]="true" [isClosable]="true" [transitionDuration]="0"
+  [size]="'normal'" class="sb-modal" appBodyScroll #modal>
   <div class="sb-modal-header">Create Group</div>
   <div class="sb-modal-content o-y-visible">
     <div class="create-group-title pb-16">Enter the following information</div>
-    <form class="sb-form">
-     
+    <form class="sb-form" [formGroup]="groupForm">
       <div class="sb-field mb-16">
         <label>Group Name</label>
-        <input class="sb-form-control is-invalid" type="text" placeholder="Enter Group Name">
-        <small class="message sb-color-error font-weight-bold">Group Name is required.</small>
+        <input class="sb-form-control" type="text" placeholder="Enter Group Name" formControlName="groupName" [ngClass]="{'is-invalid' : isFieldValid('groupName')}">
+        <small class="message sb-color-error font-weight-bold" *ngIf="isFieldValid('groupName')">Group Name is required.</small>
```

Comment:
Move all the texts to resource bundles and add telemetry events(INTERACT AND IMPRESSION)

Reply:
Resolved